docs/pages/enroll-resources/mcp-access/integration-guides/grafana.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Connect a Grafana MCP server to Teleport
+sidebar_label: Grafana
+description: Set up a Grafana MCP server for access through Teleport
+tags:
+- how-to
+- mcp
+- zero-trust
+---
+
+This guide demonstrates how to run a Grafana MCP server and connect it via
+Teleport.
+
+## How it works
+
+The [Grafana MCP server](https://github.com/grafana/mcp-grafana)
+forwards JWT tokens signed by Teleport to access Grafana and runs on a local
+endpoint reachable by Teleport. Teleport proxies all client requests to the
+server, which interacts with Grafana using permissions mapped by [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-access/configure-authentication/jwt/).
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx edition="Teleport (v18.3.0 or higher)" clients="\`tsh\` client"!)
+- Ability to configure your Grafana instance.
+- A host to run the MCP server that is reachable by the Teleport Application Service.
+- Running [Teleport Application Service](../getting-started.mdx).
```

Comment:
Is the expectation that the user will have followed the linked guide? If it is, I would say so, to make sure they're not missing anything.